torch/_dynamo/utils.py

```diff
@@ -1160,6 +1160,9 @@ def get_fake_value(node, tx):
     def fake_wrapper(e):
         if isinstance(e, torch.Tensor):
             assert isinstance(e, FakeTensor)
+            new_e = torch._prims_common.clone_preserve_strides(e)
```

Comment:
i don't know if this works because we use `storage._cdata` to test for aliasing through FakeTensor. 
```
import torch
x = torch.rand([4])

with torch._subclasses.FakeTensorMode():
    x = torch.rand([4])
    z = x.clone()

    x.data_ptr = z.data_ptr
    assert x.untyped_storage()._cdata == z.untyped_storage()._cdata
```

Reply:
Actually, all of the data ptrs of fake tensor show 0, so the above is definitely not enough to preserve aliasing

Reply:
Added a function named clone_preserve_strides_storage for cloning while preserving strides and storage.